modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/jobs/VersionMigrationsJob.kt

```diff
@@ -158,6 +159,10 @@ class VersionMigrationsJob : JobService() {
         if (previousVersionCode < 6362) {
             upgradeTrimSilenceMode()
         }
+
+        if (previousVersionCode < 9204) {
```

Comment:
Do you think we should extract this number to a variable?

Reply:
I don't think it would be helpful. It doesn't have any semantic meaning except for being used for the migration purposes. But do you have an idea for the name?